lib/opensusebasetest.pm

```diff
@@ -1331,7 +1331,7 @@ The choice is made by BACKEND and other variables.
 =cut
 
 sub select_user_serial_terminal {
-    select_serial_terminal(0);
+    select_serial_terminal(root => 0);
```

Comment:
The parameter in the function is positional. Please revert

Reply:
It's not. In the `select_serial_terminal()` there is also `$self` which makes `select_serial_terminal(0)` not work but after my change it works.

Reply:
they are positional. The first expects the instance and then the value is assigned to the second variable.

Reply:
i still dont see any reason to not fix that.

Reply:
When I revert it and call `select_user_serial_terminal()` it will log me in as **root** - not as **user**.
The `select_user_serial_terminal()` is not used anywhere else in this repository and is broken.

So can you tell me how should I fix this?

Reply:
Here is the verification run where I reverted just that one line: https://pdostal-server.suse.cz/tests/2197#step/ansible/30

Reply:
```From 591b0c115d53e4d095c0e86542ef0a81de60f4f7 Mon Sep 17 00:00:00 2001
From: Ioannis Bonatakis <ybonatakis@suse.com>
Date: Mon, 17 Oct 2022 20:05:34 +0200
Subject: Fix select_user_serial_terminal

Signed-off-by: Ioannis Bonatakis <ybonatakis@suse.com>
---
 lib/opensusebasetest.pm | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/opensusebasetest.pm b/lib/opensusebasetest.pm
index 9478fffe2..937332a6b 100644
--- a/lib/opensusebasetest.pm
+++ b/lib/opensusebasetest.pm
@@ -1331,7 +1331,8 @@ The choice is made by BACKEND and other variables.
 =cut
 
 sub select_user_serial_terminal {
-    select_serial_terminal(root => 0);
+    my ($self) = @_;
+    $self->select_serial_terminal(0);
 }
 
 =head2 upload_coredumps
-- 
2.38.0```

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15725